### DISCUSSION TITLE
Falling to update via SOUP to v2.3.21

### DISCUSSION THREAD

**Discussion by vinumsv** [Security-Onion-Solutions/securityonion#2511] [Unsupported Versions] [answered]
There is a problem downloading the so-elasticsearch:2.3.21 image. Details:

gpg: Signature made Mon 21 Dec 2020 10:46:06 PM UTC using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>"

[
![image](https://user-images.githubusercontent.com/1468742/103434570-6d890b00-4c29-11eb-9d1e-dbc60d962319.png)
](url)

Sorry if it was already discussed as i have searched for it and couldn't find

**Comment by dougburks** [comment, marked as answer]
I don't think anybody else has reported any problems like this. I just tested and was able to successfully upgrade from 2.3.10 to 2.3.21. 

Looking a little closer at your screenshot, I see what appears to be `dial tcp: i/o timeout` during the Docker image download. This could have been a transient error, so you might want to just try again.

If you continue to have issues, you'll want to ensure that you have full outbound Internet access and that you're not getting blocked by firewalls or proxy servers.

**Comment by joeb1kenobe** [comment]
TLS interception proxy has caused this error in my environment. Because of
the MitM attack performed by the proxy. Adding to manager server to the
proxy tls bypass should resolve the issue in my experience.

Joe

On Fri, Jan 1, 2021 at 6:39 AM Doug Burks <notifications@github.com> wrote:

> I don't think anybody else has reported any problems like this. I just
> tested and was able to successfully upgrade from 2.3.10 to 2.3.21. You'll
> want to ensure that you have full outbound Internet access and that you're
> not getting blocked by firewalls or proxy servers.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/2511#discussioncomment-255390>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAD6W6VVNICCBFVPW7WDEMTSXWX7JANCNFSM4VQBNA6A>
> .
>
-- 
Joe Brown

**Reply by vinumsv** [reply]
I did full reinstall and got i fixed.
